quinn/src/endpoint.rs

```diff
@@ -708,6 +713,13 @@ impl EndpointRef {
     }
 }
 
+/// Number of datagrams to allocate space for per recv
+fn recv_datagram_count(udp: &UdpState) -> usize {
```

Comment:
Style nit: move this down below remaining `EndpointRef` impls? Or maybe this should become a `UdpState` method? All the inputs are from quinn-udp...

Reply:
I'm trying to keep quinn-udp focused on the platform API, and exclude tuning parameters as much as possible.

Reply:
GRO segment count is a property of the OS. The number of worst-case datagrams we want to be able to receive in one call is a tuning decision, except that it must be a multiple of the GRO batch size if we're going to use GRO.